### PR TITLE
fix: Fix tests/util/util_spec.lua accessing undefined variable Util

### PR DESCRIPTION
**Pull request by Edmond-J-A**
fix: Fix tests/util/util_spec.lua accessing undefined variable Util
Signed-off-by: Edmond edomondja@gmail.com

### CODE REVIEW

**Comment by techoner** [comment]
@Edmond-J-A Not complete enough, please fix .

**Comment by hsluoyz** [comment]
@rushitote

File: tests/main/enforcer_spec.lua [hunk 1]
```diff
@@ -31,7 +31,7 @@ describe("Enforcer tests", function ()
         local model  = path .. "/examples/basic_model_without_spaces.conf"
         local policy  = path .. "/examples/basic_policy.csv"
 
-        local e = Enforcer:new(model, policy)
+        local e = enforcer_module:new(model, policy)
```
Comment (rushitote) [inline tests/main/enforcer_spec.lua:34]: @Edmond-J-A Please try not to change the module names, instead change the module import name at line 15 from `local enforcer_module = require("src.main.Enforcer")` to `local Enforcer = require("src.main.Enforcer")`.

**Comment by hsluoyz** [comment]
@Edmond-J-A don't flood so many commits:

image

**Comment by hsluoyz** [comment]
@Edmond-J-A ideally contains only one single commit

**Comment by hsluoyz** [comment]
@Edmond-J-A fix:

image

**Pull request closed without merging**